cg/cli/set.py

```diff
@@ -77,7 +76,8 @@ def sample(context, sex, customer, comment, downsampled_to, apptag, capture_kit,
         context.obj['status'].commit()
 
         print(click.style('update LIMS/Gender', fg='blue'))
-        lims_api.update_sample(sample_id, sex=sex)
+        if context.obj.get('lims'):
```

Comment:
Could you explain why this is behind an if statement?

Reply:
I don't know how test the LIMS part so if it is not initialized with LIMS we skip trying to send the gender to LIMS. Do you have any suggestion how we could do it?